beacon-chain/state/state-native/getters_withdrawal_test.go

```diff
@@ -367,8 +376,57 @@ func TestExpectedWithdrawals(t *testing.T) {
 		p, err := s.PendingPartialWithdrawals()
 		require.NoError(t, err)
 		require.NoError(t, s.UpdateBalancesAtIndex(p[0].Index, 0)) // This should still count as partial withdrawal.
-		_, partialWithdrawalsCount, err := s.ExpectedWithdrawals()
+		_, partialWithdrawalsCount, _, err := s.ExpectedWithdrawals()
 		require.NoError(t, err)
 		require.Equal(t, uint64(10), partialWithdrawalsCount)
 	})
+	t.Run("ExpectedWithdrawls UnitTest", func(t *testing.T) {
```

Comment:
any other good test name for here..?

Reply:
changed to `one valid withdrawal with two pending partial withdrawal`  ~